### DISCUSSION TITLE
Does HyperSerialESP32 support a relay to control LEDs on/off?

### DISCUSSION THREAD

**Discussion by sofakng** [awawa-dev/HyperSerialESP32#1] [General]
Does HyperSerialESP32 support a relay to control LEDs on/off?

This is a nice feature to ensure the LEDs are completely off and also because LEDs draw a small amount of current even when not "on".

**Comment by awawa-dev** [comment]
When you turn off the LED using for example www remote control then 1) the 'black' is sent to LED strip 2) HyperHDR stops sending next updates till the LED driver is switched on again 3) HyperSerialESP32 stops refreshing LED strip since it not received updates 4) LED power consumption is near or in fact zero. That's the software implementation. The possible new hardware solution needs custom hardware relay project that ESP could control. I am also very skeptical about this for safety reasons. Anyway it should be rather easy to signal 'no data' on some pin to external relay after HyperSerialEsp not received any data for ~5 seconds but I want to keep that project as simply as possible. Forks are always welcome.
